src/Kernel/RectorKernel.php

```diff
@@ -17,7 +17,7 @@ final class RectorKernel
     /**
      * @var string
      */
-    private const CACHE_KEY = 'v10';
+    private const CACHE_KEY = 'v13';
```

Comment:
I see you need to increase the key pretty often.

maybe we should merge https://github.com/rectorphp/rector-src/pull/3867 so we can see whether it improves DX, so changing the key is not as often required

Reply:
That PR merged 👍